light-curve/light_curve/light_curve_py/features/rainbow/_base.py

```diff
@@ -280,17 +315,22 @@ def _eval_and_get_errors(self, *, t, m, sigma, band, print_level=None):
             y=m,
             yerror=sigma,
         )
-        minuit = self.Minuit(least_squares, **initial_guesses)
+        minuit = self.Minuit(least_squares, name=self.names, **initial_guesses)
```

Comment:
That's to ensure the limits and initial parameters are being applied in correct order. Our function uses `*params` in its arguments list, so Minuit cannot properly introspect it to understand the parameter names. I guess it then falls back to just the order, and it may fail if you switch places of the parameters in limits/initial guesses dicts. So better just directly tell it what is the expected names of parameters